README.md

```diff
@@ -1,2 +1,17 @@
-# microverse-portfolio
-My first portfolio at Microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Sa Aung Htet Nyein's Microverse Portfolio
+
+> My personal portfolio and the first project of Module 1 at Microverse.
+
+## Built With
+
+- HTML, CSS, Javascript
+
+👤 **Sa Aung Htet Nyein (SASA)**
+
+- GitHub: [@aghtetnyein](https://github.com/aghtetnyein)
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly make sure your `Readme` file has all the necessary sections as seen in the [Microverse Readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided 🙏